controllers/snapshot/snapshot_adapter.go

```diff
@@ -423,7 +429,16 @@ func (a *Adapter) EnsureAllReleasesExist() (controller.OperationResult, error) {
 		gitops.SetSnapshotIntegrationStatusAsError(a.snapshot, "Failed to create new Releases: "+err.Error())
 		a.logger.LogAuditEvent("Snapshot integration status marked as Invalid. Failed to create new Releases",
 			a.snapshot, h.LogActionUpdate)
-		return controller.RequeueOnErrorOrStop(a.client.Status().Patch(a.context, a.snapshot, patch))
+		er := a.client.Status().Patch(a.context, a.snapshot, patch)
+		if er != nil {
+			a.logger.Error(er, "Failed to mark snapshot integration status as invalid",
+				"snapshot.Name", a.snapshot.Name)
+			return controller.RequeueWithError(er)
+		}
+		if clienterrors.IsForbidden(err) {
```

Comment:
Why this specific error is considered to be non-recoverable?

Reply:
I think this forbidden error need some operation to make it work again, e.g., the quota limitation is reached.

Reply:
I think that quota limit should be kept in reconciliation, so once old objects are emptied, it will recover with processing.

Reply:
But users might notice the issue several days later, at that time we might have stopped the reconciliation after failing several times.

Reply:
but if they recover, they would need to wait one day

Reply:
So let's remove this stop and see users notice this limitation generally then handle it in [STONEINTG-729](https://issues.redhat.com/browse/STONEINTG-729)